docs/admin/monitoring.rst

```diff
@@ -165,25 +165,81 @@ NodeStatus can be accessed using the JMX MBean object name
++-------------------------+---------------------------------------------------+
+| ``NodeName``            | Provides the human friendly name of the node.     |
++-------------------------+---------------------------------------------------+
+| ``ClusterStateVersion`` | Provides the version of the current applied       |
+|                         | cluster state.                                    |
++-------------------------+---------------------------------------------------+
+| ``ShardStats``          | Statistics about the number of shards located on  |
+|                         | the node.                                         |
++-------------------------+---------------------------------------------------+
+| ``ShardInfo``           | Detailed information about the shards located on  |
+|                         | the node.                                         |
++-------------------------+---------------------------------------------------+
+
+``ShardStats`` returns a `CompositeData`_ object containing statistics about
+the number of shards located on the node with the following attributes:
+
++-------------------+---------------------------------------------------------+
+| Name              | Description                                             |
++===================+=========================================================+
+| ``Total``         | The number of shards located on the node.               |
++-------------------+---------------------------------------------------------+
+| ``Primaries``     | The number of primary shards located on the node.       |
++-------------------+---------------------------------------------------------+
+| ``Replicas``      | The number of replica shards located on the node.       |
++-------------------+---------------------------------------------------------+
+| ``Unassigned``    | The number of unassigned shards in the cluster. If the  |
+|                   | role of the node is ``MASTER`` this will show the total |
+|                   | number of unassigned shards in the cluster, otherwise 0.|
++-------------------+---------------------------------------------------------+
+
+``ShardInfo`` returns an Array of `CompositeData`_ objects containing detailed
+information about the shards located on the node with the following attributes:
+
++-------------------+---------------------------------------------------------+
+| Name              | Description                                             |
++===================+=========================================================+
+| ``Id``            | The shard id. This shard id is managed by the system,   |
+|                   | ranging from 0 up to the number of configured shards of |
+|                   | the table.                                              |
++-------------------+---------------------------------------------------------+
+| ``Table``         | The name of the table this shard belongs to.            |
```

Comment:
We may also need the partition_ident for partitioned tables to distinguish between partitions.